tests/common/Configuration.cs

```diff
@@ -543,7 +552,7 @@ public static void AssertXcodeSupports32Bit ()
 
 		public static bool XcodeSupports32Bit {
 			get {
-				return Version.Parse (XcodeVersion).Major < 10;
+				return Version.Parse (XcodeVersionString).Major < 10;
```

Comment:
can't it re-use the already parsed version string ?

Reply:
Yes, let me change that.